tools/build/third_party/spirv-cross.BUILD

```diff
@@ -21,6 +21,11 @@ cc_library(
     ]),
     hdrs = [
         "spirv_glsl.hpp",
+        "spirv_hlsl.hpp",
+    ],
+    defines = [
+        "SPIRV_CROSS_C_API_GLSL",
+        "SPIRV_CROSS_C_API_HLSL",
```

Comment:
This adds these defines to everyone that depends on this. Since they are not used in the header files, use `local_defines` instead.